algorithms/max-flow/max-flow.tex

```diff
@@ -0,0 +1,82 @@
+\section{Max Flows}\index{graph!Max Flows}
+#ifdef hackpackpp
+There are several types of problems that involve finding maximum flow in a graph.
+Many of these problems can easily tackled by making small transformations to the graph.
```

Comment:
This Max-Flow section does not give a comprehensive overview of problem types, related vocabulary (sources and sinks), or why the following unrelated suggestions are important or useful. It reads more like notes taken during a lecture than a learning resource.